docs/event-types.mdx

```diff
@@ -77,12 +95,189 @@ svix event-type create '{ "name": "user.signup", "description": "A user has sign
 <TabItem value="curl">
 
 ```shell
+export SVIX_AUTH_TOKEN='AUTH_TOKEN'
+
 curl -X POST "https://api.svix.com/api/v1/event-type/" \
     -H  "Accept: application/json" \
     -H  "Content-Type: application/json" \
-    -H  "Authorization: Bearer AUTH_TOKEN" \
+    -H  "Authorization: Bearer ${SVIX_AUTH_TOKEN}" \
     -d '{ "name": "user.signup", "description": "A user has signed up" }'
 ```
 
 </TabItem>
 </CodeTabs>
+
+
+## Bulk creating event types
+
+The easiest way to bulk-create event types is by just writing a simple script to load a [`TSV` file](https://en.wikipedia.org/wiki/Tab-separated_values) or a `JSON` file with the event types and make the API requests.
+
+Here is an example `TSV` file (without headers) of events:
+
+```tsv
+user.created	A user has been created
+user.removed	A user has been removed
+user.changed	A user has changed
+```
+
+Here are some example scripts for processing the above file:
+
+<CodeTabs>
+<TabItem value="js">
+
+```js
+import { Svix } from "svix";
+import fs from "fs";
+import readline from "readline";
+
+const svix = new Svix("AUTH_TOKEN");
+
+async function execute() {
+  const fileStream = fs.createReadStream("./data.tsv");
+
+  const data = readline.createInterface({
+    input: fileStream,
+    crlfDelay: Infinity
+  });
+
+  for await (const lineItr of data) {
+    const line = lineItr.split("\t");
+    const eventType = await api.eventType.create({
+      name: line[0],
+      description: line[1],
+    });
+  }
+}
+
+execute();
+```
+
+</TabItem>
+<TabItem value="py">
+
+```python
+import json
+from svix import Svix
+from svix.openapi_client.models import EventTypeIn
+
+svix = Svix("AUTH_TOKEN")
+
+with open("./data.tsv", "r") as f:
+    for line in f:
+        name, description = line.split("\t")
+        app = svix.event_type.create(EventTypeIn(
+            description,
+            name,
```

Comment:
While testing I noticed the python generator generates these models with the arguments in alphabetical order. Pretty gross, we should look into fixing this.  I'll write a ticket.

Reply:
I didn't even notice it, because kwargs are allowed... You accidentally broke it when you changed to csv, that's why the example was wrong. Anyhow, I fixed the example now.